src/analysis/utils/tool-utils.ts

```diff
@@ -214,7 +215,13 @@ export const getToolLabelForImage = (imageId: string): ToolLabel | undefined =>
 export const getToolLabelFromFileExtension = (fileName: FileExtension): ToolLabel =>
   extensionToToolMap[getExtension(fileName)];
 
-export const getToolLabelFromCloudEnv = (cloudEnv: Runtime | App): ToolLabel => cloudEnv?.labels?.tool as ToolLabel;
+export const getToolLabelFromRuntime = (cloudEnv: Runtime): ToolLabel => cloudEnv?.labels?.tool as ToolLabel;
+export const getToolLabelFromApp = (cloudEnv: App): ToolLabel => cloudEnv?.appType as ToolLabel;
```

Comment:
ah nice. There's a few of these left over where we erroneously used the fact that two strings are often similar enum values when in the end they are distinct sets. For example, we use `getDisplayRuntimeStatus` on `AppStatus` in a few places. 🤦